test/util/test_c_mutator.py

```
@@ -0,0 +1,278 @@
+
+
+# ---------------------------------------------------------------------------
+# Helpers
+# ---------------------------------------------------------------------------
+
+def _mutant_replacements(mutants: list[Mutant]) -> set[tuple[str, str]]:
+    """Return the (original, replacement) pairs from a list of mutants."""
+    return {(m.original, m.replacement) for m in mutants}
+
+
+def _get_function(file: str, name: str):
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `_get_function`

(ANN202)